src/client/game/progress.js

```diff
@@ -21,18 +21,18 @@ export class Progress extends Component {
         return (
             <div className={style['timers']}>
                 <div>{this.props.answering ? this.timer(this.props.answerTimerValue, this.props.gameParams.answerTime) : null}</div>
-                <div>{this.props.submission ? this.timer(this.props.submissionTimerValue, this.props.gameParams.submitTime) : null}</div>
+                <div>{this.props.progressBarVisible ? this.timer(this.props.submissionTimerValue, this.props.gameParams.submitTime) : null}</div>
             </div>
         )
     }
 }
 
 const mapStateToProps = state => ({
     answering: state.game.answering,
-    submission: state.game.submission,
     answerTimerValue: state.game.answerTimerValue,
     submissionTimerValue: state.game.submissionTimerValue,
-    gameParams: state.game.gameParams
+    gameParams: state.game.gameParams,
+    progressBarVisible: state.game.progressBarVisible
```

Comment:
the progressBarVisible prop was tested, the names were bad because the submission prop was in fact not even used